src/index.html

```diff
@@ -22,6 +22,431 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region Header -->
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          class="logo__img"
+          src="./images/logo.png"
+          alt="Moyo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
```

Comment:
The `data-qa` attribute is used here for the navigation link 'Laptops & Computers'. Ensure that similar elements have consistent `data-qa` attributes if they are meant for testing or identification purposes.